tests/main/special-home-can-run-classic-snaps/task.yaml

```diff
@@ -36,7 +36,10 @@ restore: |
             # TODO: remove the apt key added above, but how?
             ;;
         postgres)
-            apt autoremove --purge -y postgresql
+            # the first one will fail because of ... a race with /etc/alternatives?
+            # /etc/postgresql and/or /var/lib/postgres not being empty?
+            # one of those (or both) (arbitrarily) (sometimes)
```

Comment:
```
The following packages will be REMOVED:
  libpq5* postgresql* postgresql-9.5* postgresql-client-9.5*
  postgresql-client-common* postgresql-common* postgresql-contrib-9.5*
  sysstat*
0 upgraded, 0 newly installed, 8 to remove and 77 not upgraded.
After this operation, 19.6 MB disk space will be freed.
(Reading database ... 133815 files and directories currently installed.)
Removing postgresql (9.5+173ubuntu0.2) ...
Removing postgresql-contrib-9.5 (9.5.14-0ubuntu0.16.04) ...
update-alternatives: warning: forcing reinstallation of alternative /usr/share/postgresql/9.5/man/man1/postmaster.1.gz because link group postmaster.1.gz is broken
Removing postgresql-9.5 (9.5.14-0ubuntu0.16.04) ...
Purging configuration files for postgresql-9.5 (9.5.14-0ubuntu0.16.04) ...
Dropping cluster main...
Cluster is not running.
Error: could not stop server, aborting
dpkg: error processing package postgresql-9.5 (--purge):
 subprocess installed post-removal script returned error exit status 1
Removing postgresql-client-9.5 (9.5.14-0ubuntu0.16.04) ...
Removing libpq5:amd64 (9.5.14-0ubuntu0.16.04) ...
Removing postgresql-common (173ubuntu0.2) ...
Removing 'diversion of /usr/bin/pg_config to /usr/bin/pg_config.libpq-dev by postgresql-common'
Purging configuration files for postgresql-common (173ubuntu0.2) ...
dpkg: warning: while removing postgresql-common, directory '/etc/postgresql' not empty so not removed
dpkg: warning: while removing postgresql-common, directory '/var/log/postgresql' not empty so not removed
dpkg: warning: while removing postgresql-common, directory '/var/lib/postgresql' not empty so not removed
Removing postgresql-client-common (173ubuntu0.2) ...
Purging configuration files for postgresql-client-common (173ubuntu0.2) ...
Removing sysstat (11.2.0-1ubuntu0.2) ...
Purging configuration files for sysstat (11.2.0-1ubuntu0.2) ...
Processing triggers for libc-bin (2.23-0ubuntu10) ...
Processing triggers for man-db (2.7.5-1) ...
Errors were encountered while processing:
 postgresql-9.5
E: Sub-process /usr/bin/dpkg returned an error code (1)
```
(ideas welcome)